### PR TITLE
DOC: update code style for user guide for #36777

### PR DESCRIPTION
**Pull request by karasinski**
Addresses part of #36777
Ran blacken-tools and checked for warnings from flake8-rst for 5 additional doc files.

### CODE REVIEW

File: doc/source/user_guide/io.rst [hunk 1]
```diff
@@ -3310,10 +3310,10 @@ applications (CTRL-V on many operating systems). Here we illustrate writing a
...
 .. code-block:: python
 
-    >>> df = pd.DataFrame({'A': [1, 2, 3],
-    ...                    'B': [4, 5, 6],
-    ...                    'C': ['p', 'q', 'r']},
-    ...                   index=['x', 'y', 'z'])
+    >>> df = pd.DataFrame({"A": [1, 2, 3],
+    ...                    "B": [4, 5, 6],
+    ...                    "C": ["p", "q", "r"]},
+    ...                   index=["x", "y", "z"])
```
Comment (dsaxton) [inline doc/source/user_guide/io.rst:3316]: Can you use black-style line wrapping here?

Comment (karasinski) [reply doc/source/user_guide/io.rst]: Added an additional commit below to address this, how does that look?

Comment (dsaxton) [reply doc/source/user_guide/io.rst]: Looks good to me, thanks

**Review by dsaxton — APPROVED** [review]

**Comment by dsaxton** [comment]
Thanks @karasinski

**Pull request merged**